### PR TITLE
refactor flutter driver command extensions

### PR DESCRIPTION
**Pull request by ufraaan**
This PR refactors the `StubNestedCommandExtension` and `StubProberCommandExtension` classes, optimizing code structure. 

Improves code readability and maintainability without altering the functionality of the classes.

### CODE REVIEW

**Comment by 12138zhen** [comment]
Where did you remove the unnecessary async/await keywords

**Comment by Ufraan** [comment]
> Where did you remove the unnecessary async/await keywords

yep, you're right, there aren't any async/await keywords in this code. maybe i got confused by another code snippet. this code has changes only in syntactic sugar for simplification, using arrow functions (`=>`) for concise one-liners.

thanks!

File: packages/flutter_driver/test/src/real_tests/stubs/stub_command_extension.dart [hunk 1]
```diff
@@ -23,9 +23,8 @@ class StubNestedCommandExtension extends CommandExtension {
   }
 
   @override
-  Command deserialize(Map<String, String> params, DeserializeFinderFactory finderFactory, DeserializeCommandFactory commandFactory) {
-    return StubNestedCommand.deserialize(params, finderFactory);
-  }
+  Command deserialize(Map<String, String> params, DeserializeFinderFactory finderFactory, DeserializeCommandFactory commandFactory) =>
+    StubNestedCommand.deserialize(params, finderFactory);
```
Comment (nate-thegrate) [inline packages/flutter_driver/test/src/real_tests/stubs/stub_command_extension.dart:27]: This looks quite a bit nicer!

Unfortunately, the Flutter repo has a [style guideline](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods) specifically for this situation:

> only use `=>` when everything, including the function declaration, fits on a single line.

Comment (moroii69) [reply packages/flutter_driver/test/src/real_tests/stubs/stub_command_extension.dart:27]: > This looks quite a bit nicer!
> 
> Unfortunately, the Flutter repo has a [style guideline](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods) specifically for this situation:
> 
> > only use `=>` when everything, including the function declaration, fits on a single line.

thank you for the info.

**Comment by nate-thegrate** [comment]
Does @ImgBotApp have an option to output the images in `UTF-8` format?

**Comment by goderbauer** [comment]
From https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8747233314619005921/+/u/run_test.dart_for_analyze_shard_and_subshard_None/stdout:

> In particular, images and other binaries must not be checked into this repository.

Can you revert the changes to the PNGs? We do not allow new images (or changes to existing images) in this repository to keep the repo size small.

Many other checks are also failing on this change. Can you take a look at those and address them? Thank you!

**Comment by goderbauer** [comment]
Actually, I am going to close this one. The syntax change is against our style guide and we don't want new image files in this repo.

**Pull request closed without merging**